build.gradle

```diff
@@ -11,6 +11,7 @@ repositories {
 }
 dependencies {
   compile 'redis.clients:jedis:[2.7.2,3)'
+  compile 'org.apache.commons:commons-lang3:3.4'
```

Comment:
In general, I like to do a format like I did with `Commons-Pool2`: lower bound of the current revision and upper bound (exclusive) of the next "safe" revision. For the "safe" revision, I usually do the next minor version up, but the Apache Commons are pretty stable in their versioning scheme, so I'd do the next major version up for them.